pkg/portal/portal.go

```diff
@@ -261,21 +265,40 @@ func (p *portal) unauthenticatedRoutes(r *mux.Router) {
 }
 
 func (p *portal) aadAuthenticatedRoutes(r *mux.Router) {
-	for _, name := range AssetNames() {
-		if name == "v1/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
-			continue
-		}
+	var names []string
 
-		if name == "v2/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/v2").HandlerFunc(p.indexV2)
-			continue
+	err := filepath.Walk(".", func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			log.Fatal(err)
+			return nil
```

Comment:
```suggestion
			return err
```

This will exit and we will exit in the error handling block. it's better to only have one place where we crash the program ()